pyfr/ctypesutil.py

```diff
@@ -13,18 +13,19 @@ def find_libc():
         return ctypes.util.find_library('c')
 
 
-def load_library(name):
+def load_library(name, **kwargs):
```

Comment:
No need for these changes.  Not that they're bad, but unused features are untested features.